Assets/MixedRealityToolkit/Inspectors/Profiles/MixedRealityPointerProfileInspector.cs

```diff
@@ -58,76 +61,65 @@ protected override void OnEnable()
+            {
+                return;
+            }
 
-            if (MixedRealityInspectorUtility.CheckMixedRealityConfigured(true, !RenderAsSubProfile))
+            if (!MixedRealityToolkit.Instance.ActiveProfile.IsInputSystemEnabled)
             {
-                if (DrawBacktrackProfileButton("Back to Input Profile", MixedRealityToolkit.Instance.ActiveProfile.InputSystemProfile))
-                {
-                    return;
-                }
+                EditorGUILayout.HelpBox("No input system is enabled, or you need to specify the type in the main configuration profile.", MessageType.Error);
+                return;
             }
 
-            CheckProfileLock(target);
+            bool wasGUIEnabled = GUI.enabled;
+            GUI.enabled = wasGUIEnabled && !CheckProfileLock((BaseMixedRealityProfile)target);
+
             serializedObject.Update();
             currentlySelectedPointerOption = -1;
 
             EditorGUILayout.Space();
-            showPointerProperties = EditorGUILayout.Foldout(showPointerProperties, "Pointer Settings", true);
-            if (showPointerProperties)
+            EditorGUILayout.LabelField("Gaze Settings", EditorStyles.boldLabel);
             {
-                using (new EditorGUI.IndentLevelScope())
-                {
-                    EditorGUILayout.PropertyField(pointingExtent);
-                    EditorGUILayout.PropertyField(pointingRaycastLayerMasks, true);
-                    EditorGUILayout.PropertyField(pointerMediator);
-
-                    EditorGUILayout.Space();
-                    showPointerOptionProperties = EditorGUILayout.Foldout(showPointerOptionProperties, "Pointer Options", true);
-                    if (showPointerOptionProperties)
-                    {
-                        using (new EditorGUI.IndentLevelScope())
-                        {
-                            pointerOptionList.DoLayoutList();
-                        }
-                    }
+                EditorGUILayout.Space();
+                EditorGUILayout.PropertyField(gazeCursorPrefab);
+                EditorGUILayout.PropertyField(gazeProviderType);
+                EditorGUILayout.Space();
 
-                    EditorGUILayout.Space();
-                    showPointerDebugProperties = EditorGUILayout.Foldout(showPointerDebugProperties, "Debug Settings", true);
-                    if (showPointerDebugProperties)
-                    {
-                        using (new EditorGUI.IndentLevelScope())
-                        {
-                            EditorGUILayout.PropertyField(debugDrawPointingRays);
-                            EditorGUILayout.PropertyField(debugDrawPointingRayColors, true);
-                        }
-                    }
+                if (RenderIndentedButton("Customize Gaze Provider Settings"))
+                {
+                    // TODO: Check if inspector locked?
```

Comment:
>// TODO: Check if inspector locked? [](start = 20, length = 35)

It would be helpful to either resolve the TODO, or provide additional information on how someone else could update this (and how they would validate, etc).

Reply:
Removal. I was going to try to be smart in the future and make sure the inspector appears on selection